index.html

```diff
@@ -0,0 +1,205 @@
+     <div class="line2"></div>
+     <p class="p2">Chief Executive Officer (CEO)</p>
+  </div>
+</div>
+
+<div class="person">
+  <img class="personImage"  src="images/mayssa5.webp"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Mayssa</h2>
+   <p class="p1"> Responsible for developing and executing the company's marketing strategies. </p>
+   <div class="line2"></div>
+   <p class="p2">Chief Marketing Officer (CMO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/santiago3.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Santiago</h2>
+   <p class="p1">Manages the company's financial operations, including budgeting, financial planning, and financial reporting..</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Financial Officer (CFO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/catherine4.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Catherine</h2>
+   <p class="p1">Oversees the day-to-day operations of the company, ensuring efficient and effective processes across various departments.</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Operations Officer (COO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/stellar1.jpeg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Stellar</h2>
+   <p class="p1">Responsible for aligning technology initiatives with business objectives, managing IT infrastructure, and driving digital transformation.</p>
+   <div class="line2"></div>
+   <p class="p2">Chief Technology Officer (CTO)</p>
+</div>
+</div>
+
+<div class="person im">
+  <img class="personImage"  src="images/houda6.jpg"alt="Mohamme_image">
+<div class="clm2">
+  <h2 class="name">Houda</h2>
+   <p class="p1">Oversees all aspects of human resources, including recruitment, employee relations, training and development, and HR policies. </p>
+   <div class="line2"></div>
+   <p class="p2">Human Resources Director</p>
+</div>
+</div>
+
+<div id="more" class="more">MORE
+  <img src="images/arrow_down 1.png" alt="arrow-icon">
+</div>
+<div id="less" class="more">LESS
+  <img src="images/arrow_up.png" alt="arrow-icon">
+</div>
+</div>
+
+</section>
+<section class="section4">
+<div class="partnerLogo">
+  <h3 class="partnerHeader">Partners</h3>
+  <div id="line3"></div>
+<div>
+  <img src="images/R.png" alt="partner-logo">
+  <img src="images/bb.png" alt="partner-logo">
+  <img src="images/tt.png" alt="partner-logo">
+  <img src="images/s.png" alt="partner-logo">
+ 
+</div>
+</div>
+
+<footer class="footer">
+   <img src="images/reserve.jpeg" class="now" alt=""> <h4></h4>Skincare indulgence </h4>
+   <div class="foodiv">
+   <h3 class="fooh3"><a href=""> Call to Join</h3>
+   </a><p class="foopara">
+    Reserve Your Spot Now to Indulge in Skincare
+   </p>
+  </div>
+</footer>
+</section>
+
+</main>
+
+
+
+<script src="main.js"></script>
+</body>
+</html>
```

Comment:
Please implement the following changes for the mobile view of your `home` page :

1.  Kindly fix the popup screen of the `see more` link of your ` main activities`

![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/844a015d-2850-4c8c-ace3-03f948e3ab6f)

2. Please remove the partners and footer section for the home page of the mobile view 
3. kindly align the `founders`section properly as yours is center which isn't same with the design

Your design :
![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/cd2086ac-b9ed-41dd-b823-df4203250080)

Design given :
![image](https://github.com/Nessrine88/Capstone-Project/assets/20537712/820f6ce8-4479-4741-8400-42d6d7cc4f61)